src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,101 @@
-import React from 'react';
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
 import './FindMovie.scss';
-
+import { findMoviesFromOMDb } from '../../api/findMovies';
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+export const FindMovie = ({ addMovie }) => {
+  const [movieTitle, findMovieTitle] = useState('');
+  const [notFound, setNotFound] = useState(false);
+  const [foundMovie, previewFoundMoive] = useState('');
```

Comment:
```suggestion
  const [foundedMovie, setFoundedMovie] = useState(null);
```
use null as default value